packages/code-infra/src/cli/cmdBuild.mjs

```diff
@@ -134,6 +134,12 @@ async function writePackageJson({ packageJson, bundles, outputDir, cwd, addTypes
   delete packageJson.devDependencies;
   delete packageJson.imports;
 
+  if (packageJson.private === false) {
```

Comment:
It makes sense to warn before the build starts rather than after its finished since at that point, `build` folder will have files but no `pacakge.json`.

Reply:
Ok, I have moved the logic to run earlier. Makes sense, I cram that logic in the very first place I could spot 😆.